

## Next-Generation Sequencing (NGS)

For most of the courses in this section, some basic experience of using a UNIX/LINUX command line and some R (or Python) knowledge are assumed and essential. 

### General

- [**ManchesterBCF**](https://www.bmh.manchester.ac.uk/research/facilities/bioinformatics/) &mdash; 
[Introduction to NGS analysis](https://www.youtube.com/playlist?list=PLLwM58CflFb-Cv0BQv-2XaY4-z-vq1-8l)
	> Video Recordings

- [**EMBL-EBI**](https://www.ebi.ac.uk/training/) &mdash; Functional genomics 
	- [I: Introduction and designing experiments](https://www.ebi.ac.uk/training/online/courses/functional-genomics-i-introduction-and-design/)
	- [II: Common technologies and data analysis methods](https://www.ebi.ac.uk/training/online/courses/functional-genomics-ii-common-technologies-and-data-analysis-methods/)
	- [III: Submitting your data and functional genomics databases](https://www.ebi.ac.uk/training/online/courses/functional-genomics-iii-submitting-data/)


- [**SIB**](https://www.sib.swiss/) &mdash; [ NGS: Quality control, Alignment, Visualisation](https://sib-swiss.github.io/NGS-introduction-training/) | [GitHub](https://github.com/sib-swiss/NGS-introduction-training)

- [**Harvard Chan Bioinformatics Core**](https://bioinformatics.sph.harvard.edu/) &mdash;  [ Accessing genomic reference and experimental sequencing data](https://hbctraining.github.io/Accessing_public_genomic_data/) | [GitHub](https://github.com/hbctraining/Accessing_public_genomic_data)

- [**CRUK CI**](https://www.cruk.cam.ac.uk/)  &mdash; [ Workshop on Functional Genomics - summer school 2020](https://bioinformatics-core-shared-training.github.io/cruk-summer-school-2020/) | [GitHub](https://github.com/bioinformatics-core-shared-training/cruk-summer-school-2020)


### RNA-seq



- [**ManchesterBCF**](https://www.bmh.manchester.ac.uk/research/facilities/bioinformatics/) &mdash; [Guide to RNA-Seq Analysis](https://ycl6.gitbook.io/guide-to-rna-seq-analysis/)



- [**CRUK CI**](https://www.cruk.cam.ac.uk/)  &mdash; [ Introduction to Bulk RNA-seq data analysis](https://bioinformatics-core-shared-training.github.io/Bulk_RNAseq_Course_Apr22/) | [GitHub](https://github.com/bioinformatics-core-shared-training/Bulk_RNAseq_Course_Apr22)

- [**Harvard Chan Bioinformatics Core**](https://bioinformatics.sph.harvard.edu/) &mdash; [ Introduction to bulk RNA-seq](https://hbctraining.github.io/Intro-to-rnaseq-hpc-salmon-flipped/schedule/links-to-lessons.html) | [GitHub](https://github.com/hbctraining/Intro-to-rnaseq-hpc-salmon-flipped)


- [**WEHI**](https://www.wehi.edu.au/) &mdash; [A Tidy Transcriptomics introduction to RNA sequencing analyses](https://stemangiola.github.io/bioc_2020_tidytranscriptomics/) | [GitHub](https://github.com/stemangiola/bioc_2020_tidytranscriptomics/)


- [**Rockefeller University Bioinformatics Resource Center**](https://www.rockefeller.edu/bioinformatics/) &mdash; [RNAseq with Bioconductor](https://rockefelleruniversity.github.io/RU_RNAseq/index.html)  

- [**ISU Bioinformatics Workbook**](https://bioinformaticsworkbook.org/#gsc.tab=0) &mdash; [RNA-Seq data Analysis](https://bioinformaticsworkbook.org/dataAnalysis/RNA-Seq/RNA-SeqIntro/RNAseq-using-a-genome.html#gsc.tab=0) | [GitHub](https://github.com/ISUgenomics/bioinformatics-workbook)

- [**ISU Bioinformatics Workbook**](https://bioinformaticsworkbook.org/#gsc.tab=0) &mdash; [Differential Gene Expression analysis](https://bioinformaticsworkbook.org/dataAnalysis/RNA-Seq/RNA-SeqIntro/Differential-Expression-Analysis.html#gsc.tab=0) | [GitHub](https://github.com/ISUgenomics/bioinformatics-workbook)

- [**SIB**](https://www.sib.swiss/) &mdash; [ Introduction to RNA-Seq: From quality control to pathway analysis](https://sib-swiss.github.io/RNAseq-introduction-training/) | [GitHub](https://github.com/sib-swiss/RNAseq-introduction-training)
	> Course designed for using a dedicated computer cluster 

- [**Harvard Chan Bioinformatics Core**](https://bioinformatics.sph.harvard.edu/) &mdash; [ Differential Gene Expression Analysis](https://hbctraining.github.io/DGE_workshop_salmon_online/schedule/links-to-lessons.html) | [GitHub](https://github.com/hbctraining/DGE_workshop_salmon_online)


- [Marta Pérez Alcántara](https://gitlab.com/mperalc) &mdash; [Bulk RNA-seq tutorial](https://mperalc.gitlab.io/bulk_RNA-seq_workshop_2021/) | [GitLab](https://gitlab.com/mperalc/bulk_RNA-seq_workshop_2021/)


### ChIP-seq

- [**ManchesterBCF**](https://www.bmh.manchester.ac.uk/research/facilities/bioinformatics/) &mdash; 
[ChIP-seq Analysis Using Galaxy (2015)](https://www.youtube.com/playlist?list=PLLwM58CflFb-8YrwzXIpJIWRJiX6izIEX) 
	> Video Recordings

- [**Harvard Chan Bioinformatics Core**](https://bioinformatics.sph.harvard.edu/) &mdash; [ Introduction to ChIP-seq using high performance computing](https://hbctraining.github.io/Intro-to-ChIPseq/) | [GitHub](https://github.com/hbctraining/Intro-to-ChIPseq)

- [**CRUK CI**](https://www.cruk.cam.ac.uk/)  &mdash; [ Quantitiative analysis of ChiP-seq, ATAC-seq, and related DNA enrichment assays](https://github.com/bioinformatics-core-shared-training/Quantitative-ChIPseq-Workshop)

- [**UMass** Integrated ChIP-seq Data Analysis Workshop](https://hukai916.github.io/IntegratedChIPseqWorkshop/) | [GitHub](https://github.com/hukai916/IntegratedChIPseqWorkshop/)




### ATAC-seq

- [**UMass MCCB** Best practices for ATAC-seq QC and data analysis](https://haibol2016.github.io/ATACseqQCWorkshop/) | [GitHub](https://github.com/haibol2016/ATACseqQCWorkshop/)


### Single cell

#### 2022

- [**CRUK CI**](https://www.cruk.cam.ac.uk/)  &mdash; [ Introduction to single-cell RNA-seq data analysis](https://bioinformatics-core-shared-training.github.io/UnivCambridge_ScRnaSeqIntro_Feb2022/) | [GitHub](https://github.com/bioinformatics-core-shared-training/UnivCambridge_ScRnaSeqIntro_Feb2022)


- [**Harvard Chan Bioinformatics Core**](https://bioinformatics.sph.harvard.edu/) &mdash; [ Single-cell RNA-seq data analysis workshop](https://github.com/hbctraining/scRNA-seq_online) | [GitHub](https://hbctraining.github.io/scRNA-seq_online/schedule/links-to-lessons.html)


- [**SIB**](https://www.sib.swiss/) &mdash; [ Single-cell Transcriptomics](https://sib-swiss.github.io/single-cell-training/latest/) | [GitHub](https://github.com/sib-swiss/single-cell-training/)

#### 2021


- [**SIB**](https://www.sib.swiss/)/[**NBIS**](https://github.com/NBISweden)/[**SciLifeLab**](https://www.scilifelab.se/) &mdash; [ Advanced topics in Single Cell Omics](https://nbisweden.github.io/single-cell_sib_scilifelab_2021/) | [GitHub](https://github.com/NBISweden/single-cell_sib_scilifelab_2021)


- [**Leiden Computational Biology Center**](https://www.lcbc.nl) &mdash; [ Single Cell Analysis](https://github.com/LeidenCBC/MGC-BioSB-SingleCellAnalysis2021)


- [**Cambridge Bioinformatics Training Unit**](https://bioinfotraining.bio.cam.ac.uk/) &mdash; [ Analysis of single cell RNA-seq data](https://www.singlecellcourse.org/index.html) | [GitHub](https://github.com/hemberg-lab/scRNA.seq.course) 


- [**WEHI**](https://www.wehi.edu.au/) &mdash; [   Single cell RNA-seq analysis workshop](
https://yunshun.github.io/SingleCellWorkshop/) | [GitHub](https://github.com/yunshun/SingleCellWorkshop/)
	> Integration analysis of multi-sample single cell RNA-seq experiment

- [Marta Pérez Alcántara](https://gitlab.com/mperalc) &mdash; [ Single cell RNA-seq tutorial](https://mperalc.gitlab.io/scRNA-seq_workshop_2021/) | [GitLab](https://gitlab.com/mperalc/scRNA-seq_workshop_2021)


#### 2020

- [**SIB**](https://www.sib.swiss/) &mdash; [ Advanced topics in single-cell transcriptomics](https://github.com/fmicompbio/adv_scrnaseq_2020)


- [**NBIS**](https://github.com/NBISweden) &mdash; [ Single cell RNA-seq analysis workshop](https://nbisweden.github.io/workshop-scRNAseq/) | [GitHub](https://github.com/nbisweden/workshop-scRNAseq)



- [**Dana-Farber Cancer Institute**](https://www.dana-farber.org/) &mdash; [ Trajectory inference across conditions: differential expression and differential progression](https://kstreet13.github.io/bioc2020trajectories/) | [GitHub](https://github.com/kstreet13/bioc2020trajectories)



#### 2019

- [**ELIXIR EXCELERATE**](https://elixir-europe.org/) &mdash; [ Single RNA-seq data analysis with R](https://nbisweden.github.io/excelerate-scRNAseq/) | [GitHub](https://github.com/NBISweden/excelerate-scRNAseq)


### Spatial Transcriptomics
- [**Scanpy Tutorial**](https://scanpy.readthedocs.io/en/stable/tutorials.html#) &mdash; [Spatial Data](https://scanpy.readthedocs.io/en/stable/tutorials.html#spatial-data) 




### Genome assembly and annotation

- [**ISU Bioinformatics Workbook**](https://bioinformaticsworkbook.org/#gsc.tab=0) &mdash; [Introduction to Genome Assembly](https://bioinformaticsworkbook.org/dataAnalysis/GenomeAssembly/Intro_GenomeAssembly.html#gsc.tab=0) | [GitHub](https://github.com/ISUgenomics/bioinformatics-workbook)




### Variant analysis

- [**SIB**](https://www.sib.swiss/) &mdash; [ NGS - Genome variant analysis](https://sib-swiss.github.io/NGS-variants-training/) | [GitHub](https://github.com/sib-swiss/NGS-variants-training/)

- [**ISU Bioinformatics Workbook**](https://bioinformaticsworkbook.org/#gsc.tab=0) &mdash; [Variant Discovery](https://bioinformaticsworkbook.org/dataAnalysis/VariantCalling/variant-calling-index.html#gsc.tab=0) | [GitHub](https://github.com/ISUgenomics/bioinformatics-workbook)

- [**WEHI**](https://www.wehi.edu.au/) &mdash; [  A introduction to SNV analyses in whole genome sequencing](https://github.com/PapenfussLab/IntroductionToGenomicsWorkshop/) | [GitHub](https://papenfusslab.github.io/IntroductionToGenomicsWorkshop/)

- [**Waldron lab CUNY**](https://waldronlab.io/) &mdash; [ Copy number variation analysis with Bioconductor](https://waldronlab.io/CNVWorkshop/) | [GitHub](https://github.com/waldronlab/CNVWorkshop)



### Proteomics
- [**EMBL-EBI**](https://www.ebi.ac.uk/training/) &mdash; [ Proteomics bioinformatics](https://www.ebi.ac.uk/training/events/proteomics-bioinformatics-2021/)
	> * Materials and live recording available. 
	> * MaxQuant/Perseus tutorial: Day 3
	> * MsqRob tutorial: Day 4  (Practical on Downstream quantitative analysis/statistics)


### Enrichment analysis

- [**Waldron lab CUNY**](https://waldronlab.io/) &mdash; [ Functional enrichment analysis of high-throughput omics data](https://waldronlab.io/enrichOmics/) | [GitHub](https://github.com/waldronlab/enrichOmics)
	> microarrays and RNA-seq; DE analysis with limma, edgeR, DESeq2


- [**WEHI**](https://www.wehi.edu.au/) &mdash; [   Gene-set enrichment analysis workshop](https://davislaboratory.github.io/GenesetAnalysisWorkflow/) | [GitHub](https://github.com/DavisLaboratory/GenesetAnalysisWorkflow/)
	> pre-processed RNA-seq data; singscore package

- [**SIB**](https://www.sib.swiss/) &mdash; [ Enrichment Analysis](https://sib-swiss.github.io/enrichment-analysis-training/) | [GitHub](https://github.com/sib-swiss/enrichment-analysis-training/)
	> including ggplot for visualisation

### Pathway analysis

- [**ManchesterBCF**](https://www.bmh.manchester.ac.uk/research/facilities/bioinformatics/) &mdash;  [Using Ingenuity Pathway Analysis (IPA) at the University of Manchester](https://www.youtube.com/watch?v=KpSKd1PdrK0)
	> Video Recordings

- [**EMBL-EBI**](https://www.ebi.ac.uk/training/) &mdash; [ Reactome Quick tour](https://www.ebi.ac.uk/training/online/courses/reactome-quick-tour/)


### Public data

- [**Waldron lab CUNY**](https://waldronlab.io/) &mdash; [ Public data resources and Bioconductor](https://waldronlab.io/PublicDataResources/) | [GitHub](https://github.com/jmacdon/Bioc2020Anno)

- [**Waldron lab CUNY**](https://waldronlab.io/) &mdash; [ Multi-omic Integration of cBioPortal and TCGA data with MultiAssayExperiment](https://waldronlab.io/MultiAssayWorkshop/) | [GitHub](https://github.com/waldronlab/MultiAssayWorkshop/)

- [**James W. MacDonald, UW**](https://tools.niehs.nih.gov/srp/people/details.cfm?Person_ID=34924) &mdash;[Introduction to Bioconductor annotation resources](https://jmacdon.github.io/Bioc2020Anno/) | [GitHub](https://github.com/jmacdon/Bioc2020Anno)
	> Instructions:
	> * Option 1: Following the 'Preparation' section in the link workshop [homepage](https://jmacdon.github.io/Bioc2020Anno/) and going through the workshop through the Docker container created for this workshop.
	> * Option 2: if not using Docker, download the workshop ``.Rdata`` through github page [here](https://github.com/jmacdon/Bioc2020Anno/tree/master/inst/extdata) and also install the individual R / Bioconductor packages used in this workshop listed [here](https://jmacdon.github.io/Bioc2020Anno/articles/AnnotationWorkshop.html#r-bioconductor-packages-used). Then follow the workshop materials [here](https://jmacdon.github.io/Bioc2020Anno/articles/AnnotationWorkshop.html).








